Sources/SwiftProtobuf/Message+BinaryAdditions.swift

```diff
@@ -31,26 +31,24 @@ extension Message {
     if !partial && !isInitialized {
       throw BinaryEncodingError.missingRequiredFields
     }
```

Comment:
This should get removed, otherwise the validation is done twice because of the chaining of calls.